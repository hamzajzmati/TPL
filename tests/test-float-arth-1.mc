int main()
{
   float a = 1.5 ; 
   float b = 2.5 ; 
   printfloat( a + b);
   printfloat( a / b);
   printfloat( a * b);
   printfloat( a - b);
  return 0; 
}
