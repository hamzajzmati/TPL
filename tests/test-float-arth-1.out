4.00
0.60
3.75
-1.00

